infra-as-code/bicep/modules/hubNetworking/parameters/hubNetworking.parameters.min.json

```diff
@@ -0,0 +1,157 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "parHubNetworkAddressPrefix": {
+      "value": "10.20.0.0/16"
+    },
+    "parSubnets": {
+      "value": [
+        {
+          "name": "AzureBastionSubnet",
+          "ipAddressRange": "10.20.0.0/24"
+        },
+        {
+          "name": "GatewaySubnet",
+          "ipAddressRange": "10.20.254.0/24"
+        },
+        {
+          "name": "AzureFirewallSubnet",
+          "ipAddressRange": "10.20.255.0/24"
+        }
+      ]
+    },
+    "parDnsServerIpArray": {
+      "value": []
+    },
+    "parPublicIpSku": {
+      "value": "Standard"
+    },
+    "parBastionEnabled": {
+      "value": true
+    },
+    "parBastionSku": {
+      "value": "Standard"
+    },
+    "parDdosEnabled": {
+      "value": true
+    },
+    "parAzureFirewallEnabled": {
+      "value": true
+    },
+    "parAzureFirewallTier": {
+      "value": "Standard"
+    },
+    "parAzureFirewallAvailabilityZones": {
+      "value": []
+    },
+    "parNetworkDnsEnableProxy": {
+      "value": true
+    },
+    "parDisableBgpRoutePropagation": {
+      "value": false
+    },
+    "parPrivateDnsZonesEnabled": {
+      "value": true
+    },
+    "parPrivateDnsZones": {
+      "value": [
+        "privatelink.azure-automation.net",
+        "privatelink.database.windows.net",
+        "privatelink.sql.azuresynapse.net",
+        "privatelink.dev.azuresynapse.net",
+        "privatelink.azuresynapse.net",
+        "privatelink.blob.core.windows.net",
+        "privatelink.table.core.windows.net",
+        "privatelink.queue.core.windows.net",
+        "privatelink.file.core.windows.net",
+        "privatelink.web.core.windows.net",
+        "privatelink.dfs.core.windows.net",
+        "privatelink.documents.azure.com",
+        "privatelink.mongo.cosmos.azure.com",
+        "privatelink.cassandra.cosmos.azure.com",
+        "privatelink.gremlin.cosmos.azure.com",
+        "privatelink.table.cosmos.azure.com",
+        "privatelink.eastus.batch.azure.com",
+        "privatelink.postgres.database.azure.com",
+        "privatelink.mysql.database.azure.com",
+        "privatelink.mariadb.database.azure.com",
+        "privatelink.vaultcore.azure.net",
+        "privatelink.managedhsm.azure.net",
+        "privatelink.eastus.azmk8s.io",
```

Comment:
Why `eastus`? Do we need to add a comment or something saying this in this file incase they dont see the readme?

Reply:
This is due to copy/paste + find/replace. The bicep template contains these zones as the default value so I was bringing parameter file up to parity. However, the bicep template uses string interpolation to account for the region.

To be safe, I think it might be best to remove region specific zones from the parameters file until we can discuss a better way of handling.